_posts/release/2018-04-23-accumulo-1.9.0.md

```diff
@@ -1,39 +1,43 @@
 ---
 title: Apache Accumulo 1.9.0
-draft: true
 ---
 
-Apache Accumulo 1.9.0 is a minor release on the 1.x branch. This release would be considered a
-maintainence release on 1.8 branch except there are some API additions which resulted in a new minor
-release. Users of 1.8.x versions of Accumulo should upgrade to 1.9.0.  There will be no more bug fix
-releases on the 1.8 branch. This release contains changes from 96 issues.  See [GitHub] and [JIRA]
-for a list of changes.
+Apache Accumulo 1.9.0 is a minor release on the 1.x branch. This release would
+be considered a maintainence release on 1.8 branch except there are some API
+additions which resulted in a new minor release. Users of 1.8.x versions of
+Accumulo should upgrade to 1.9.0. There will be no more bug fix releases on the
+1.8 branch. This release contains changes for nearly a hundred issues. See
+[GitHub] and [JIRA] for a list of changes.
 
 Below are resources for this release:
 
 * [User Manual][user_manual] - In-depth developer and administrator documentation.
 * [Javadocs][javadoc] - Accumulo 1.9.0 API
-* [Examples][examples] - Code with corresponding readme files that give step by step instructions
-  for running example code.
+* [Examples][examples] - Code with corresponding readme files that give step by
+  step instructions for running example code.
 
 ## Notable Changes
 
 ### Deprecated ClientConfiguration API using commons config
 
-In [ACCUMULO-4611], public API in ClientConfiguration using commons config types was deprecated to
-better support Hadoop 3 in the future. New methods were added to replace these methods which cause
-this release to be a 1.9.0 release.  These changes allow removal of commons config from Accumulo's 
-API in 2.0.0.  If using ClientConfiguration, then switching from existing constructors to the new
-static methods [create()], [fromFile()], or [fromMap()] will ensure your code works in 2.0.0.
+In [ACCUMULO-4611], public API in ClientConfiguration using commons config
+types was deprecated to better support Hadoop 3 in the future. New methods were
+added to replace these methods which cause this release to be a 1.9.0 release.
+These changes allow removal of commons config from Accumulo's API in 2.0.0.  If
+using ClientConfiguration, then switching from existing constructors to the new
+static methods [create()], [fromFile()], or [fromMap()] will ensure your code
```

Comment:
I wanted to ensure these links worked correctly, so I built these changes locally.  The links worked.  When I clicked on them I saw the following in the terminal running jekyll

```
[2018-04-23 15:28:22] ERROR `/1.9/apidocs/resources/fonts/dejavu.css' not found.
```

Reply:
Shouldn't be a problem, but it is fixed in newer Java https://bugs.openjdk.java.net/browse/JDK-8093199